README.md

```diff
@@ -122,20 +122,21 @@ Comments
 **Example:**
 ```objc
 /*
- * This is a multi-line comment. The opening and closing markers are on their
- * own lines, and each other line is preceded by a * that is indented by one
- * space.
- *
- * This is a new paragraph in the same block comment.
+ This is a multi-line comment. The opening and closing markers are on their
+ own lines, and each other line is preceded by a * that is indented by one
+ space.
+
+ This is a new paragraph in the same block comment.
  */
 
 stop(); // Hammer-time!
 
 // this is a very brief comment.
 ```
 
-* Use [doxygen](http://www.stack.nl/~dimitri/doxygen/manual/docblocks.html)-style comments for all the headers. Make
-sure to use the available markup tags like `@param`, `@return`, etc. The `///` form is preferred for single line
+* Document all methods and properties in the headers using Xcode8's documentation style. You can select a property
+or method and use `Cmd`-`/` to see this in action. Make sure to use the available markup tags like `@param`,
+`@return`, etc. (these will be auto-generated for you by Xcode8). The `///` form is preferred for single line
```

Comment:
How about omitting the Xcode version number here. Since it was made clear in the sentences just before. Also gets old in just a few months :)